packages/nextcloud/pubspec.yaml

```diff
@@ -2,13 +2,13 @@ name: nextcloud
 version: 1.0.0
 
 environment:
-  sdk: '>=2.19.0 <3.0.0'
+  sdk: '>=3.0.0 <4.0.0'
 
 dependencies:
-  cookie_jar: ^3.0.1
+  cookie_jar: ^4.0.2
   crypto: ^3.0.2
   crypton: ^2.0.5
-  intl: ^0.17.0
+  intl: any
```

Comment:
Is there a reason we still need this now?
I think pinning it to 0.18.0 might be good.

Reply:
If another flutter app would try to import the package but still uses flutter <3.10 (requiring intl 0.17) then it would fail. I think the best is to unpin it, as much as I hate doing it.

Reply:
That's the magic of unreleased software hehe